### DISCUSSION TITLE
How can I get a CheckBox in Excel using OpenXML?

### DISCUSSION THREAD

**Discussion by JaneConan** [OfficeDev/Open-XML-SDK#1847] [General]
We can get the Value by Excel

![image](https://github.com/dotnet/Open-XML-SDK/assets/26090153/12cbd975-ea90-473c-a43a-1f74e71c9cf4)

BT,

we cannot find in OpenXML 
by
`C#
var controls = wsPart.Worksheet.Descendants<DocumentFormat.OpenXml.Spreadsheet.Control>();
`

![image](https://github.com/dotnet/Open-XML-SDK/assets/26090153/7b72d83d-d2c5-47d5-88f1-f85814c32cb3)

**Comment by AlfredHellstern** [comment]
This is more of a discussion rather than an issue. Please also post this on Stackoverflow using the [openxml-sdk] label,  https://stackoverflow.com/questions/tagged/openxml-sdk

**Comment by mikeebowen** [comment]
Hi @JaneConan, I see a list of checkboxes that your code finds. What is it that you cannot find? Also, if possible, please share the file your are opening and your sample code to repro.
